features/create_a_copy.feature

```diff
@@ -3,6 +3,7 @@ Feature: Create a copy
   I want to create a new session from an existing session
   So that I can avoid typing everything from scratch again
 
+@wip
```

Comment:
Why was this added and can it be removed?

Reply:
The preview state will no longer load as it is built from react components which have been moved to start fresh.